Snowflake.Data/Logger/SFLoggerFactory.cs

```diff
@@ -55,6 +80,30 @@ public static SFLogger GetLogger<T>()
                 return new SFLoggerEmptyImpl();
             }
         }
-    }
 
+        internal static ILogger GetCustomLogger<T>()
+        {
+            // If true, return the default/specified logger
+            if (s_isCustomLoggerEnabled)
+            {
+                // If no logger specified, use the default logger: Microsoft's console logger
+                if (s_customLogger == null)
+                {
+                    ILoggerFactory factory = LoggerFactory.Create(
+                        builder => builder
+                        .AddConsole()

```

Comment:
does it mean that we create a custom logger with logging to console always enabled?
Could we defer the decision about appenders to the customer configuration?

Reply:
This means if a customer enables custom logging without setting their own logger then a console logger will be created

If a customer sets the logger then it will return their custom logger with their chosen appenders (logging to console is not enabled unless they added it themselves)